src/webgl.js

```diff
@@ -1024,17 +1021,18 @@ export function main(player, canvas, options) {
     return;
   }
 
-  const ext =
-    gl.getExtension('OES_texture_float') ||
-    gl.getExtension('MOZ_OES_texture_float') ||
-    gl.getExtension('WEBKIT_OES_texture_float');
-  gl.getExtension('OES_texture_half_float') ||
-  gl.getExtension('MOZ_OES_texture_half_float') ||
-  gl.getExtension('WEBKIT_OES_texture_half_float');
+  // enable WebGL extensions
   gl.getExtension('EXT_color_buffer_float');
   gl.getExtension('OES_texture_float_linear');
   gl.getExtension('OES_texture_half_float_linear');
 
+  // set default clearing values
+  gl.clearColor(0.0, 0.0, 0.0, 1.0); // Clear to black, fully opaque
+  gl.clearDepth(1.0); // Clear everything
+  gl.enable(gl.DEPTH_TEST); // Enable depth testing
+  gl.depthFunc(gl.LEQUAL); // Near things obscure far things
```

Comment:
I think you can turn off depth testing, since we're only dealing with a single orthogonal plane.

Reply:
Turning it off seems to slow down the rendering by 1 - 2ms on my machine. We must have something that the DEPTH_TEST optimizes away.